### DISCUSSION TITLE
Are house numbers supposed to be shown on map?

### DISCUSSION THREAD

**Discussion by timothywashere** [ENT8R/streetcomplete-mapstyle#118] [Q&A] [answered]
#82 has been closed and I infer from this that house numbers should now be shown for areas ?

Because I don't see them. Or have I misunderstood?

**Comment by matkoniecz** [comment]
Yes, house numbers should be shown.

Maybe something goes wrong with areas that are not buildings or someone tagged addresses as `name`?

Can you check that

- you zoomed in
- https://www.openstreetmap.org/ maps shows mapped addresses

?

If no, can you give an example of affected location?

**Comment by timothywashere** [comment]
I've been looking around the UK and can't see anywhere with house numbers, even London which is very detail mapped.

(Details removed)

Will post a screenshot of SC from my phone

**Comment by timothywashere** [comment, marked as answer]
Here they are
![Screenshot_20211201-121645](https://user-images.githubusercontent.com/87654968/144233082-9fbeec3b-3120-4034-acca-366b89f87b46.png)

It's the same no matter how far I zoom in

**Reply by matkoniecz** [reply]
Do you see street names? 

Do you have Android 12?

Then it is a known bug in Tangram that results in no displaying labels in general (I failed to find it, but it is reported already upstream)

**Reply by timothywashere** [reply]
> 

Yes, Android 11, no street names also.
Checked my old Android 9 phone, labels appear fine.

Thanks for the info!

**Reply by mnalis** [reply]
I think this was the issue https://github.com/streetcomplete/StreetComplete/issues/3448 which links to upstream Tangram issue https://github.com/tangrams/tangram-es/issues/2303

**Reply by timothywashere** [reply]
> I think this was the issue [streetcomplete/StreetComplete#3448](https://github.com/streetcomplete/StreetComplete/issues/3448) which links to upstream Tangram issue [tangrams/tangram-es#2303](https://github.com/tangrams/tangram-es/issues/2303)

Interesting. My phone is a Pixel also, Pixel 3.
Wish I hadn't upgraded, Android 12 is annoying me in so many ways

**Reply by westnordost** [reply]
What is annoying?

Matt Blair already fixed the issue, he is looking for people (with Android 12) to reproduce that his fix is working correctly.

**Reply by timothywashere** [reply]
Android 12 in general is full of bugs. Nothing else that directly impacts SC. The worst is that The app switcher stops working and you have to restart. Generally unresponsive to user input. If it was Apple I would suspect they were trying to force people to buy a new phone.

**Reply by westnordost** [reply]
Maybe it's only borken on the Pixels then ;-)
